ci/tasks/build-java-lambdas.yml

```diff
@@ -0,0 +1,56 @@
+container_limits: {}
+image_resource:
+  type: registry-image
+  source:
+    repository: governmentdigitalservice/pay-concourse-runner-with-java-21
+caches:
+  - path: src/.m2
+inputs:
+  - name: src
+outputs:
+  - name: build
+params:
+  app_name:
+platform: linux
+run:
+  path: ash
+  args:
+  - -ec
+  - |
+    #!/bin/ash
```

Comment:
Can this be moved to a .sh file so it is to shellcheck and better IDE support?

Reply:
Is there an existing pattern to do this? All the task files in the `ci/tasks` directory have the ash/bash script in the yml file.

Reply:
https://github.com/alphagov/pay-ci/blob/a233afcad90d2c8f1d89ea19ac939091012ef1c8/ci/pipelines/deploy-to-staging.yml#L3662

^ is node though. 

bash could follow similar pattern. One in pkl https://github.com/alphagov/pay-ci/blob/f06052e1757da0ecfd365006753690728e8da998/ci/pkl-pipelines/common/shared_resources_for_deploy_pipelines.pkl#L119

Reply:
There's an example in this also open PR: https://github.com/alphagov/pay-ci/pull/1163/files#diff-21cd77e1d1e1ab24276cc0488b99256c3499af730656130b6f8c902293c35ac8R16